### DISCUSSION TITLE
How to disable this tool and get back to the old bios?

### DISCUSSION THREAD

**Discussion by apcbat** [DavidS95/Smokeless_UMAF#5] [Q&A] [answered]
I tried this on my laptop with AMD R7 6800H. It didn‘t work when I try to disable Modern Standby, and caused other problems even I reset all the options to the default.  

Is there any way to get back to the old bios?

**Comment by DavidS95** [comment, marked as answer]
This does not replace your old bios, this edits values within ur normal bios with a gui, either u didnt reset all the settings or clearing cmos didnt reset those options. So there is no way to disable this tool because there isnt anything that was enabled or disabled.

**Comment by hdchieh** [comment]
> I tried this on my laptop with AMD R7 6800H. It didn‘t work when I try to disable Modern Standby, and caused other problems even I reset all the options to the default.
> 
> Is there any way to get back to the old bios?
@wjsir 
I have the same problem. have you fix it ?

**Reply by apcbat** [reply]
Sorry，I haven't fix it, I'm waiting for new version of ROM. hope it should solve the problem.

I tried to enter to the default bios and press F9 to reset options. But my laptop still pops green screen sometimes when I close the lid of my laptop and reopen it after a while.  you can try it.

I'm not sure if this tool changes default bios options but I'm sure the problem above not exists before I use this tool.

**Reply by hdchieh** [reply]
I encountered a blue screen while attempting to boot up my laptop, but, finally, I solved the problem by starting my PC in safe mode. Somehow, it's back to normal after that.

**Reply by DavidS95** [reply]
Bios clear, so reset the bios as u normally would either with a cmos clear taking out the battery or take out the laptops internal battery.  Note this tool only shows hidden bios settings whether they work or not is out of my control. Alot of these settings do not work and were never meant to which is why they are not displayed by default. This does not replace the normal bios, this edits values within the normal bios but that is it. Returning to the old bios should be as easy as unplugging ur bootable usb and doing it as you have always done it.

**Reply by grotesque** [reply]
@DavidS95 
Does resetting BIOS to default factory settings by pressing F2 and resetting the settings (_not opening laptop and removing battery etc_.) removes changes made by this tool?

Also thanks for your work!

**Reply by Espionage724** [reply]
> Does resetting BIOS to default factory settings by pressing F2 and resetting the settings (_not opening laptop and removing battery etc_.) removes changes made by this tool?

I changed "Above 4G Decode" and a WoL setting for Ethernet and they were kept after a 30-sec power button CMOS reset on Dell Latitude 5591. Other options reset fine.
